File: Backend/src/main/java/ssafy/d210/backend/dto/response/certificate/CertificateResponse.java
package ssafy.d210.backend.dto.response.certificate;

import lombok.*;

import java.time.LocalDate;
import java.util.Date;

//
@Getter
@Builder
public class CertificateResponse {
    private long lectureId;
    private String title;
    private String categoryName;
    private Integer certificate;
    // userlecture entity    private LocalDate certificateDate; 에 따른 데이터 형식 수정
    private Date certificateDate;

    // native query 사용시 필요
    public CertificateResponse(long lectureId, String title, String categoryName, Integer certificate, Date certificateDate) {
        this.lectureId = lectureId;
        this.title = title;
        this.categoryName = categoryName;
        this.certificate = certificate;
        this.certificateDate = certificateDate;
    }
}

File: Android/app/src/main/java/com/example/second_project/data/model/dto/response/CertificateResponse.kt
package com.example.second_project.data.model.dto.response

data class CertificateResponse(
    val timeStamp : String,
    val code : Int,
    val status : String,
    val data : List<CertificateData>
)

data class CertificateData(
    val lectureId: Int,
    val title: String,
    val categoryName: String,
    val certificate: Int,
    val certificateDate: String
)

